app/Enums/RoleName.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Enums;
+
+enum RoleName: int
```

Comment:
These should be strings like the PermissionName enum. You can also reference the values of both enums in the database seeders.

Additionally, you can create a method that will look something like this:

```
public function label(): string
{
    return match($this) {
        self::SUPER_ADMIN => 'Super Admin',
        self::PROJECT_OWNER => 'Project Owner',
        self::MEMBER => 'Member',
    }
}
```

A similar method called `label` will be useful for the `PermissionName` enum as well.

Reply:
implemnted